src/main/java/ru/spbstu/controller/VcfController.java

```diff
@@ -27,4 +31,22 @@
             return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body("Incorrect vcf file");
         }
     }
+
+    @PostMapping("/getAnnotatedVariants")
+    @ResponseBody
+    public AnnotatedVariantList getAnnotatedVariants(@RequestBody VariantList requestedVariantList) {
+        List<Variant> requestedVariants = requestedVariantList.getVariants();
+        List<Variant> annotatedVariants = new ArrayList<>();
+
+        for (Variant var: requestedVariants) {
```

Comment:
Can we use Stream instead?

Reply:
Modified for-loop List items processing into Stream processing.